pkgs/misc/emulators/wine/base.nix

```diff
@@ -15,9 +15,24 @@ let
   prevName = name;
   prevPlatforms = platforms;
   prevConfigFlags = configureFlags;
+  setupHook = makeSetupHook {
+    name = "darwin-mingw-hook";
+    substitutions = {
+      darwinSuffixSalt = stdenv.cc.suffixSalt;
+      mingwGccsSuffixSalts = map (gcc: gcc.suffixSalt) mingwGccs;
+    };
+  } ./setup-hook.sh;
```

Comment:
Maybe name this file `setup-hook-darwin.sh`, to make directory listing more useful.

Reply:
Arguably the variable name here could also include Darwin

Reply:
I’ve updated the PR to do both. Thanks for the feedback!